src/app/tasks/create-task/create-task.component.ts

```diff
@@ -35,18 +35,26 @@ export class CreateTaskComponent {
 
   onSubmit () {
     // Get filled out form data using form group
+
+    let taskType : Option | undefined = this.taskTypes.find( taskType => 
```

Comment:
there shouldnt be a case where taskType or taskStatus dont exist as theyre mandatory fields

Reply:
same as the comment above about revising